common/cfn/notification.yaml

```diff
@@ -417,4 +421,25 @@ Resources:
       Threshold: 0
       TreatMissingData: notBreaching
 
+  NewsstandSentAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions: [!Ref AlarmTopic]
+      AlarmDescription: !Sub
+          - triggers if less that ${DailyNewsstandPushCount} sent for ${Stage} in the last 24 hours.
```

Comment:
- triggers if less that ${DailyNewsstandPushCount} daily edition notification was sent for ${Stage} in the last 24 hours.

Reply:
Yes so less than 1 for PROD, less than 0 for CODE

Reply:
No I mean if you expand the sentence without the variable name, you lose the context:
>triggers if less that 1 sent for PROD in the last 24 hours.

I'm suggesting you add context such that the description includes what we're monitoring

>triggers if less that 1 daily edition notification was sent for PROD in the last 24 hours.

Reply:
I might be confused, but is less than 24 hours too strict?
If they're early one day and late the next  (maybe editorial is held up) then it'd be more than 24 hours?
Daylight savings will also push this to 25 hours once a year.
Perhaps this could be 26hours .

Reply:
That's a good point, also the daily edition might be published late, so 26 to 28 hours sounds about right

Reply:
Changed to 28 and updated description

Reply:
@alexduf @markrichardsg 28 hours isn't a goer. It breaks in cfn : "

  | Metrics  cannot be checked across more than a day (EvaluationPeriods * Period  must be <= 86400) (Service: AmazonCloudWatch; Status Code: 400; Error  Code: ValidationError; Request ID:  0bbdd85a-3372-11e9-a0f6-7dcbb9aa765a)"

I'm having a think about what else could work ..

Reply:
Maybe you could put the `Period` to 1h, and the `EvaluationPeriods` to 28? I have no idea if that would work